alpharaw/ms_data_base.py

```diff
@@ -25,7 +32,7 @@ class MSData_Base:
     """
     Spectrum dataframe containing the following columns:
 
-    - `rt` (float64): in minutes
+    - `rt` (float64): in minutes. `rt_sec` will be RT in seconds.
```

Comment:
? `rt_sec` is not part of the interface?

Reply:
No. Only `rt`